CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+### Contributing ###
+
+Thank you for your interest in `strong-error-handler`, an open source project
+administered by StrongLoop.
```

Comment:
administered by loopbackio ?

Reply:
I think it's meant to be administered by an entity, and replacing it with `loopbackio` seems a bit odd. A few more suggestions:

- Thank you for your interest in <repo name>.
- Thank you for your interest in <repo name>, an open source project.
- Thank you for your interest in <repo name>, an open source project as part of LoopBack.
WDYT?

Reply:
> Thank you for your interest in , an open source project as part of LoopBack.

I think it is better.